staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -240,7 +240,7 @@ func (d *Helper) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodF
 				select {
 				case <-ctx.Done():
 					// return here or we'll leak a goroutine.
-					returnCh <- fmt.Errorf("error when evicting pod %q: global timeout reached: %v", pod.Name, globalTimeout)
+					returnCh <- fmt.Errorf("error when evicting pod %q in namespace %q: global timeout reached: %v", pod.Name, pod.Namespace, globalTimeout)
```

Comment:
```suggestion
					returnCh <- fmt.Errorf("error when evicting pod %q: global timeout reached: %v", fmt.Sprintf("%s/%s", pod.Namespace, pod.Name), globalTimeout)
```

There is precedent for that at
https://github.com/kubernetes/kubernetes/blob/3aa59f7f3077642592dc8a864fcef8ba98699894/staging/src/k8s.io/kubectl/pkg/drain/filters.go#L82